packaging/machine-learning-api.spec

```diff
@@ -314,6 +315,7 @@ HTML pages of lcov results of ML API generated during rpm build
 %define enable_tizen_feature_check -Denable-tizen-feature-check=false
 %define enable_ml_service_check -Denable-ml-service=false
 %define enable_gcov -Denable-gcov=false
+%define enable_nntrainer -Denable-nntrainer=false
```

Comment:
How about default is true, and at line 336
%if 0%{?gcov}
%define enable_gcov -Denable-gcov=true
%define enable_nntrainer -Denable_nntrainer = false
%endif